python_modules/dagster/dagster/_core/definitions/utils.py

```diff
@@ -132,6 +132,11 @@ def validate_group_name(group_name: Optional[str]) -> str:
     if group_name:
         check_valid_chars(group_name)
         return group_name
+    elif group_name == "":
+        raise DagsterInvalidDefinitionError(
+            "You have passed an empty string for group_name."
```

Comment:
We generally avoid second person in error messages, so I would change the first line to something like. "Empty asset group name was provided, which is not permitted."

Reply:
fixed this issue, thanks for the review, @sryza !